manifests/vanilla/vsphere-csi-driver.yaml

```diff
@@ -148,6 +148,7 @@ data:
   "improved-volume-topology": "true"
   "block-volume-snapshot": "false"
   "csi-windows-support": "false"
+  "decouple-with-cpi": "false"
```

Comment:
Can we change the feature name as "use-csi-nodeid"?
"decouple-with-cpi" is giving a wrong indication that CSI is independent of CPI and we do not need CPI at all?

Reply:
`use-csinode-id` is used now.